examples/bruteforce_nbody/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
 add_executable(bruteforce_nbody bruteforce_nbody.cpp)
-target_link_libraries(bruteforce_nbody hipSYCL)
+target_link_libraries(bruteforce_nbody)
```

Comment:
You can remove this line entirely - in the current formulation, it just says, for target `bruteforce_nbody` link with libraries in `<empty set>`. If we are at it, maybe we should think about doing a proper cmake integration using `find_package(hipSYCL)` and `add_sycl_to_target()`. I can give you some examples if you are interested.

Reply:
Thank you, I'd like to do a proper CMake integration, so can you give me some examples?